openpoiservice/server/__init__.py

```diff
@@ -19,6 +20,13 @@
 basedir = os.path.abspath(os.path.dirname(__file__))
 ops_settings = yaml.safe_load(open(os.path.join(basedir, 'ops_settings.yml')))
 
+geocoder = None
+geocode_categories = None
+if ops_settings['geocoder'] is not None:
+    geocoder = GeocoderSetup(list(ops_settings['geocoder'].items())[0]).define_geocoder()
+    geocode_categories = CategoryTools('categories.yml').generate_geocode_categories()
+
+
 if "TESTING" in os.environ:
```

Comment:
Here you have to decide if you replace the strings. I think this is a one time call so just leave it like that.